examples/python_side_metrics.py

```diff
@@ -39,7 +39,12 @@
 
 metric_registry = CollectorRegistry()
 multiprocess.MultiProcessCollector(metric_registry)
-counter = Counter("inference_result", "statistic of result", ("status", "worker_id"))
+counter = Counter(
+    "inference_result",
+    "statistic of result",
+    ("status", "worker_id"),
+    registry=metric_registry,
```

Comment:
If the registry parameter is not specified, the Counter will not be monitored by Prometheus.